opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -140,4 +147,14 @@ public SearchScrollRequest scrollRequest() {
   public void reset() {
     scrollId = null;
   }
+
+  /**
+   * Convert a scroll request to string that can be included in a cursor.
+   * @return a string representing the scroll request.
+   */
+  @Override
+  public String toCursor() {
+    // TODO: probably should serialize exprValueFactory here as well.
```

Comment:
TODO: probably?

Reply:
Having an `OpenSearchStorageEngine` instance we rebuild it every time. If we change the design and stop using `StorageEngine`, we have to serialize the factory as well.